client/components/post.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export default class Post extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      timeAgo: null,
+      likes: [],
+      comments: []
+    };
+  }
```

Comment:
You can remove this whole constructor for now, since it's not used for anything.